scripts/deploy.sh

```diff
@@ -78,6 +79,12 @@ if [[ "${DEPLOY_AIO}" == "yes" ]]; then
     # set network speed for vms
     echo "net_max_speed: 1000" >>$RPCD_VARS
 
+    # Copy ironic AIO config if ironic is enabled
+    if [[ "$DEPLOY_IRONIC" == "yes" ]]; then
+      cp -a -a ${OA_DIR}/etc/openstack_deploy/conf.d/ironic.yml.aio /etc/openstack_deploy/conf.d/ironic.yml
```

Comment:
Thanks for the review @mrda.  Somehow your review was against an "outdated diff".  The extra flag was removed in the last PR.